src/librbd/librbd.cc

```diff
@@ -1377,6 +1377,27 @@ namespace librbd {
     return r;
   }
 
+  int RBD::group_image_snap_list(IoCtx& group_ioctx, const char *group_name,
+          const char *group_snap_name, std::string &group_snap_id,
+          std::vector<group_image_snap_info_t> *snaps,
+          size_t group_snap_info_size)
+  {
+    TracepointProvider::initialize<tracepoint_traits>(get_cct(group_ioctx));
+    tracepoint(librbd, group_snap_list_enter,
```

Comment:
The tracepoint name should be `group_image_snap_list_enter` and you should add it to `tracing/librbd.tp`, but actually we gave up adding LTTNG traces to new functions. So I recommend to just remove this.